src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.macos.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Runtime.Versioning;
+
+namespace CommunityToolkit.Maui.Core.Platform;
+
+[UnsupportedOSPlatform("MacCatalyst"), UnsupportedOSPlatform("MacOS")]
```

Comment:
I don't think we support macOS

Reply:
Correct - we don't.

I did this just because I was unsure how robust/smart the `UnsupportedOSPlatform` is. 

I.e., I was unsure if would it would generate a compiler warning if only `UnsupportedOSPlatform("MacCatalyst")` was listed.

Reply:
our target framework is maccatalyst, so MacOS is not necessary